src/Xrefcheck/Util/Interpolate.hs

```diff
@@ -0,0 +1,54 @@
+{- SPDX-FileCopyrightText: 2018-2019 Serokell <https://serokell.io>
+ -
+ - SPDX-License-Identifier: MPL-2.0
+ -}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+module Xrefcheck.Util.Interpolate where
```

Comment:
Let's add an explicit export list. `stripTrailingSpaces` and `stripLastNewline` are not part of the public API of this module.

(And also a link to the notes I mentioned in my other comment)

```suggestion
module Xrefcheck.Util.Interpolate
  ( -- $notes
    interpolateIndentF
  , interpolateBlockListF
  , interpolateBlockListF'
  )
  where
```